cob_twist_controller/include/cob_twist_controller/limiters/limiter.h

```diff
@@ -35,145 +36,203 @@
 
 #define LIMIT_SAFETY_THRESHOLD 0.1/180.0*M_PI
 
+/* BEGIN LimiterJointContainer *******************************************************************************/
+/// Joint Container for limiters, implementing interface methods.
+class LimiterJointContainer : public LimiterJointBase
```

Comment:
in fact, I think we do not need two different/separate containers
maybe we could let the `LimiterContainer` derive from both `LimiterJointBase` and `LimiterCartesianBase`
`class LimiterContainer : public LimiterJointBase, public LimiterCartesianBase`
thus providing both 
`virtual KDL::JntArray enforceLimits(const KDL::JntArray& q_dot_ik, const KDL::JntArray& q) const;` **and**
`virtual KDL::Twist enforceLimits(const KDL::Twist& v_in) const;`